metastats-infrastructure/src/test/groovy/life.qbic.metastats/OpenBisSearchSpecification.groovy

```diff
@@ -1,58 +1,80 @@
 package life.qbic.metastats
 
-import life.qbic.metastats.datamodel.MetaStatsSample
-import life.qbic.metastats.request.OpenBisSearch
-import life.qbic.metastats.request.OpenBisSession
+
 import spock.lang.Specification
 
-class OpenBisSearchSpecification extends Specification{
+class OpenBisSearchSpecification extends Specification {
+
+    /*   OpenBisSession session
+
+       def setup(){
+           def file = new File("/Users/jnnfr/private/credentials.json.properties")
```

Comment:
change hardcoded file save location!

Reply:
Right, in the file are user credentials therefore I cannot be provided for automated testing, I deleted the test.

Reply:
fdfdd99